cubi-tools/update_workflow.py

```diff
@@ -14,7 +15,16 @@ def main():
     print(f"Project directory set as: {project_dir}")
     ref_repo_clone = args.ref_repo_clone
```

Comment:
my IDE tells me that this variable is never used - what does it do?

Reply:
The whole section has been modified.